cmd/web3/main.go

```diff
@@ -245,7 +245,9 @@ func main() {
 					EnvVar: pkVarName,
 				},
 			},
-			Action: start,
+			Action: func(c *cli.Context) error {
+				return start(ctx, c)
```

Comment:
What's the benefit of this change?

Reply:
The ctx is already cancelled on signal interrupts.

Reply:
Really?  Since when?

Reply:
Not by default. It is already set up in main: https://github.com/gochain-io/web3/blob/master/cmd/web3/main.go#L43-L52

Reply:
Ahh, I see, didn't catch that.